test/unit/services/test_catalogue_item.py

```diff
@@ -1327,19 +1473,30 @@ def test_update_add_non_mandatory_property(
         CatalogueItemIn(
             catalogue_category_id=catalogue_item.catalogue_category_id,
             manufacturer_id=catalogue_item.manufacturer_id,
-            **FULL_CATALOGUE_ITEM_A_INFO,
+            **{
+                **FULL_CATALOGUE_ITEM_A_INFO,
+                "created_time": catalogue_item.created_time,
+            },
         ),
     )
     assert updated_catalogue_item == catalogue_item
 
 
 def test_update_remove_non_mandatory_property(
```

Comment:
Is this test working properly? `catalogue_item` has 2 props but 3 are provided in `assert_called_once_with`

Reply:
Sorry by props I meant `properties`. `catalogue_item`, which is used to assert against the `updated_catalogue_item`, only has two `properties` but three `properties` are provided to the `assert_called_once_with` for the `update` repo method. If three `properties` are sent to the database then I am assuming three should be returned so `catalogue_item` should have three?

Reply:
Yes that does make sense to me due to https://github.com/ral-facilities/inventory-management-system-api/pull/174 - its adding in the non mandatory one back in but with a value of `None`. The order on line 1512/1513 should be reversed though considering its a dictionary.